pkg/auth/manager.go

```diff
@@ -18,6 +18,8 @@ import (
 	"github.com/cilium/cilium/pkg/policy"
 )
 
+const authSignalBackoffTime = time.Second
```

Comment:
Could you add a comment with some rough context about why one second seems about the right amount of time? Is this corresponding to default TCP retry period?

Reply:
If we're unsure about the value, it can be an option to expose a hidden flag for this and just avoid documenting / avoid exposing in helm, just in case we might want to change it later in a production environment. But if we're pretty confident about the underlying logic for this time then that might not be necessary.